contrib/seeds/makeseeds.py

```diff
@@ -39,16 +39,16 @@
     r"0.16.(0|1|2|3|99)|"
     r"0.17.(0|0.1|1|2|99)|"
     r"0.18.(0|1|99)|"
-    r"0.19.(0|1|2|99)|"
+    r"0.19.(0|1|2|99).(0|1)|"
     r"0.20.(0|1|2|99)|"
     r"0.21.(0|1|2|99)|"
     r"22.(0|1|99).0|"
-    r"23.(0|1|99).0|"
+    r"23.(0|1|2|99).0|"
     r"24.(0|1|2|99).(0|1)|"
     r"25.(0|1|2|99).0|"
-    r"26.(0|1|99).0|"
-    r"27.(0|1|99).0|"
-    r"28.(0|99).0|"
+    r"26.(0|1|2|99).0|"
+    r"27.(0|1|2|99).0|"
+    r"28.(0|1|99).0|"
```

Comment:
We shouldn't have `|` at the end of the last clause, as this will make it match the empty string too (so effectively everything starting with `Satoshi:` matches).

Reply:
> We shouldn't have `|` at the end of the last clause, as this will make it match the empty string too (so effectively everything starting with `Satoshi:` matches).

Good catch. It looks like this would mean that the regex has been doing this since a8ec9eede4c745c6b6fd76974816ffad8034129a one year ago?

Reply:
Yes, i didn't notice before ! It's too easy to make this mistake.

It also makes me wonder whether this is a useful filter at all, or whether to replace it with a simpler one that extracts the version number, followed by some minimum/maximum bounds check. But that's not something that needs to be addressed in this PR.

Reply:
Thanks!